library/src/main/java/com/chuckerteam/chucker/api/Chucker.kt

```diff
@@ -30,6 +37,20 @@ public object Chucker {
             .setFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
     }
 
+    internal fun createShortcut(context: Context) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N_MR1) {
+            context.getSystemService(ShortcutManager::class.java)?.let {
+                val shortcut = ShortcutInfo.Builder(context, SHORTCUT_ID)
+                    .setShortLabel(context.getString(R.string.chucker_shortcut_label))
+                    .setLongLabel(context.getString(R.string.chucker_shortcut_label))
+                    .setIcon(Icon.createWithResource(context, R.mipmap.chucker_ic_launcher_round))
+                    .setIntent(getLaunchIntent(context).setAction(Intent.ACTION_VIEW))
+                    .build()
+                it.addDynamicShortcuts(listOf(shortcut))
```

Comment:
This also can throw and should be handled here. Look at the [docs](https://developer.android.com/reference/android/content/pm/ShortcutManager#addDynamicShortcuts(java.util.List%3Candroid.content.pm.ShortcutInfo%3E)).

Reply:
But I see it already been catched.
![image](https://user-images.githubusercontent.com/10363352/112593292-3d28c580-8e42-11eb-8aea-aac7aab9729e.png)

Reply:
Catched